src/tools/orb_radar.rs

```diff
@@ -291,4 +334,24 @@ impl OrbRadar {
             OrbSource::Chest => state.settings.color_orb_chests,
         }
     }
+
+    fn collected_orbs(state: &mut AppState) -> HashSet<i32> {
+        let world = state
+            .noita
+            .as_mut()
+            .and_then(|n| n.get_world_state().unwrap_or(None));
+        let Some(world) = world else {
+            tracing::warn!("Could not read noita.get_world_state()");
+            return HashSet::new();
+        };
+
+        let Ok(collected_orbs) = world
+            .orbs_found_thisrun
+            .read_storage(state.noita.as_mut().unwrap().proc())
+        else {
+            tracing::warn!("Could not read world.orbs_found_thisrun");
+            return HashSet::new();
+        };
+        collected_orbs.iter().cloned().collect()
```

Comment:
i32 could use `copied` (doesn't matter, just semantics lol)
But also remember what I said about vec of at most 36 ints being very likely to be faster than hashset (not like we care, but an additional bonus of not having to convert it here to just slow it down lmao)

hm in fact you could convert the orb id list into a bitset stored as a single u64 :joy:

eh actually the only thing of allthat I really care about is `cloned` -> `copied` for some reason lmao